src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/adyen-checkout.js

```diff
@@ -110,12 +122,15 @@ function initializeBillingEvents() {
           );
         },
         onCancel: (data, component) => {
+          paypalTerminatedEarly = false;
```

Comment:
same here, maybe I just don't get the logic behind it :)

Reply:
Hi @cyattilakiss , a bit late to reply but...
Please, see the problem described in the original ticket.
https://github.com/Adyen/adyen-salesforce-commerce-cloud/issues/403

If the PayPal window is terminated by the user, there is no way to detect this in the checkout.

Best,
Pavel

Reply:
Hi @pavelkukov,

this was fixed here https://github.com/Adyen/adyen-salesforce-commerce-cloud/pull/411

Regards,
Zaid
Adyen

Reply:
Thank you, @zaiddreakh !

Reply:
@zaiddreakh , I see the fix in develop branch only. When will be released?